src/csharp/Intel/Iced.UnitTests/Intel/AssemblerTests/AssemblerTests64.cs

```diff
@@ -97,6 +97,20 @@ public sealed partial class AssemblerTests64 {
 		}
 #endif
 
+
+		[Fact]
+		public void TestDeclareData_db_array() {
+			Assert.Throws<ArgumentNullException>(() =>
+				TestAssemblerDeclareData(c => c.db(null), new byte[] { })
```

Comment:
This doesn't verify that the thrown exception is from calling `db()`.

Here's an example. This should also be in its own test method. It tests some other condition than the other test cases in this method.

```suggestion
var c = new Assembler(xxxxxxx);
Assert.Throws<ArgumentNullException>(() => c.db(null));
```